pkg/configmap/services/services.go

```diff
@@ -14,6 +14,49 @@ func init() {
 	ServicesConfigMapName = servicesConfigMapName()
```

Comment:
Consider squashing this commit with the previous one. (https://github.com/openshift/windows-machine-config-operator/pull/1004/commits/f523a455c16269b5304ea47d1b1b3c534c7888f9)